base/docs/basedocs.jl

```diff
@@ -1,740 +1,660 @@
+is compiled directly rather than requiring a runtime `eval()` call. Macro arguments may
+include expressions, literal values, and symbols. For example:
+
+    macro sayhello(name)
+        return :( println("Hello, ", \$name) )
+    end
+
+This macro takes one argument: `name`. When `@sayhello` is encountered, the quoted
+expression is expanded to interpolate the value of the argument into the final
+expression.
+"""
+kw"macro"
+
+"""
+`importall` imports all names exported by the specified module, as if `import` were used
+individually on all of them.  For example:
+
+    importall Distributions
 
-        bar(x) = 2x
-        foo(a::MyType) = bar(a.x) + 1
-        show(io, a::MyType) = print(io, "MyType \$(a.x)")
+As with `import`, functions imported by `importall` can be extended.
+"""
+kw"importall"
+
+"""
+`local` introduces a new local variable. For example:
+
+    function foo(n)
+        x = 0
+        for i = 1:n
+            local x
+            x = i
         end
+        x
+    end
+
+    julia> foo(10)
+    0
+
+Here `local x` introduces a separate `x` inside the loop, so the function returns `0`.
+"""
+kw"local"
+
+"""
+`global x` makes `x` in the current scope and its inner scopes refer to the global
+variable of that name.   In the example below, `global` is needed so the function can
+modify the global variable `z`:
+
+    z=3
+    function foo()
+        global z=6
+    end
+
+    julia> foo()
+    6
+    julia> z
+    6
+
+Without the `global` declaration in `foo()`, a new local variable would have been
+created inside foo(), and the `z` in the global scope would have remained equal to `3`.
+"""
+kw"global"
+
+"""
+`let` statements allocate new variable bindings each time they run. Whereas an
+assignment modifies an existing value location, `let` creates new locations. This
+difference is only detectable in the case of variables that outlive their scope via
+closures. The `let` syntax accepts a comma-separated series of assignments and variable
+names:
+
+    let var1 = value1, var2, var3 = value3
+        code
+    end
+
+The assignments are evaluated in order, with each right-hand side evaluated in the scope
+before the new variable on the left-hand side has been introduced. Therefore it makes
+sense to write something like `let x = x`, since the two `x` variables are distinct and
+have separate storage.
+"""
+kw"let"
+
+"""
+`quote` creates multiple expression objects in a block without using the explicit `Expr`
+constructor. For example:
+
+    ex = quote
+        x = 1
+        y = 2
+        x + y
+    end
+
+Unlike the other means of quoting, `:( ... )`, this form introduces `QuoteNode` elements
+to the expression tree, which must be considered when directly manipulating the tree.
+For other purposes, `:( ... )` and `quote .. end` blocks are treated identically.
+"""
+kw"quote"
+
+"""
+`'` is the conjugate transposition operator:
+
+    > A = reshape(1:4, 2,2)
```

Comment:
Looks like they've been like that since they were added back in #13217. I've replaced them with `julia>`.